libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -894,36 +894,49 @@ bool AP_AHRS::use_compass(void)
 // return the quaternion defining the rotation from NED to XYZ (body) axes
 bool AP_AHRS::get_quaternion(Quaternion &quat) const
 {
+    // backends always return in XYZZY frame; rotate result according to trim
```

Comment:
```suggestion
    // backends always return in XYZ frame; rotate result according to trim
```

Pre-existing: saying frame XYZ isn't super useful.  Which is why FRD is usually used. The XYZ part is presumed, but now you get info. on positive directions like in NED.

Reply:
Nope, that really was supposed to be `XYZZY` frame here.  In hindsight perhaps I should have used the more traditional `FOOBAR` to avoid actually using those magic three letters!

I left this because I'm not absolutely sure what we are supposed to call that frame in ArduPilot.  I *think* it is "autopilot frame".  So this function takes something in the autopilot frame and rotates it into the vehicle frame.

I would suggest "FRD" is definitely wrong in that I would always assume that is vehicle frame.

If people can confirm that "autopilot frame" is correct, I'll update the description here before we have to hash it out on the call :-)

Reply:
I did google XYZZY and that came up but it didn't make sense given the context. It does now! I see weird frame names and I ask questions :)

Oh sorry this is the rotation from the INS, ahrs, autopilot, blah to the vehicle body. I assumed it was the vehicle body frame because above it says,
`// return the quaternion defining the rotation from NED to XYZ (body) axes`

That was a bad assumption. In this file most everything that says just `body` means autopilot body not vehicle body. A bit confusing as in some places body does mean vehicle instead in this file as well. 

In that case `autopilot_FRD` frame could work? Or one of INS, IMU, AHRS _FRD since we can accept external navigation now.

Reply:
I've changed this to `autopilot-XYZ`